demos/2_Iterative/03_PDHG.ipynb

```diff
@@ -1040,20 +1040,20 @@
     "\n",
     "# Plot middle line profile\n",
     "show1D([pdhg_tv_explicit.solution,pdhg_tv_implicit_regtk.solution], slice_list=[('horizontal_y',int(ig2D.voxel_num_y/2))],\n",
-    "       label = ['TV (explicit)','TV (implicit)'], title='Middle Line Profiles')\n"
+    "       dataset_labels = ['TV (explicit)','TV (implicit)'], title='Middle Line Profiles')\n"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "a0f699e0",
+   "id": "55",
    "metadata": {},
    "source": [
```

Comment:
In ex3 I think we should set nonnegativity to True

With it False we see artifacts in the background:
![image](https://github.com/user-attachments/assets/fc39d827-0d0c-4b3a-acf0-dbad2a377d0d)


With it true:
![image](https://github.com/user-attachments/assets/613878ef-596d-4705-95ae-2bdf3f3e36d3)

Additionally, in the text above we mention this constraint and give an example of setting it:
![image](https://github.com/user-attachments/assets/ecc424c0-f86a-4fef-a118-b606faa53cb7)


Now this makes the solutions the same so we may need to edit this text :

![image](https://github.com/user-attachments/assets/20938698-2210-49ad-8892-8aaf8b2550a3)